test/cases/testdata/replacen/test-replacen-0374.yaml

```diff
@@ -14,3 +14,31 @@ cases:
   want_result:
   - x:
     - This is &lt;b&gt;HTML&lt;/b&gt;!
+- data: {}
+  modules:
+  - |
+    package generated
```

Comment:
💭 What's the standard for test cases? "generated" isn't true anymore here...

Reply:
I've been using `package test` in most places. We might be able to just migrate all of the existing test cases to use that. Here's the hello-world example that I've pointed people at: https://github.com/open-policy-agent/opa/blob/master/test/cases/testdata/helloworld/test-helloworld-1.yaml